.github/workflows/integration-tests-on-release.yaml

```
@@ -18,7 +18,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        kubernetes-versions: ["v1.19.16", "v1.21.14", "v1.23.10", "v1.25.0"]
+        kubernetes-versions: ["v1.21.14", "v1.21.14", "v1.23.10", "v1.26.0"]
```

How about these versions for the release builds.
```suggestion
        kubernetes-versions: ["v1.21.14", "v1.23.15", "v1.24.9", "v1.26.0"]
```

---

done